sakura_core/window/CWnd.h

```diff
@@ -119,9 +119,9 @@ class CWnd
 	void DestroyWindow();
 
 private: // 2002/2/10 aroka アクセス権変更
-	HINSTANCE	m_hInstance;	// アプリケーションインスタンスのハンドル
-	HWND		m_hwndParent;	// オーナーウィンドウのハンドル
-	HWND		m_hWnd;			// このダイアログのハンドル
+	HINSTANCE	m_hInstance = nullptr;	// アプリケーションインスタンスのハンドル
+	HWND		m_hwndParent = nullptr;	// オーナーウィンドウのハンドル
+	HWND		m_hWnd = nullptr;		// このダイアログのハンドル
 #ifdef _DEBUG
 	WCHAR		m_szClassInheritances[1024];
```

Comment:
これは若干アヤシイですが、こうしないと m_szClassInheritances の in-class initialize ができないと思います。

```suggestion
	WCHAR		m_szClassInheritances[1024]{};
```
コンストラクターで wcscpy しちゃうので「初期化漏れ」にはならないんですけどね。